modules/service/src/test/scala/lucuma/odb/graphql/OdbSuite.scala

```diff
@@ -305,8 +307,16 @@ abstract class OdbSuite(debug: Boolean = false) extends CatsEffectSuite with Tes
         FakeItcVersions.pure[IO]
     }
 
-  // override in tests that need an http client
-  protected def httpRequestHandler: Request[IO] => Resource[IO, Response[IO]] = _ => Resource.eval(IO.pure(Response.notFound[IO]))
+  protected val httpClientResource: Resource[IO, Client[IO]] =
+    for 
+      tctx <- TLSContext.Builder.forAsync[IO].insecureResource
```

Comment:
Some JDKs (like mine) can't verify JPL's cert, so I added this hack to turn off cert checks.